examples/llama.android/app/src/androidTest/java/com/example/llama/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.llama
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.ext.junit.runners.AndroidJUnit4
+
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
+    @Test
+    fun useAppContext() {
+        // Context of the app under test.
+        val appContext = InstrumentationRegistry.getInstrumentation().targetContext
+        assertEquals("com.example.llama", appContext.packageName)
```

Comment:
probably we don't need this test since it doesn't test anything llama.cpp related

Reply:
I'll think about a few basic tests and add them to the branch for your review. Sound good?